README.md

```diff
@@ -12,56 +12,70 @@ Since the very beginning, the Ada language has been formally defined through a p
 
-The ARG performs most of its work in terms of Ada Interpretations (AIs), which are documents that describe an identified issue with the Ada Reference Manual, or which propose enhancements to the Ada programming language.  Such issues may be identified within the ARG itself, or may be identified by the wider Ada community.  Historically the ada-comment mailing list has been used by members of the Ada community to identify problems or suggest enhancements.  
+Language Study Groups proposed by the community must be approved by the ARG. Once approved, those proposing the creation of the study group can work towards the creation of a new AI for formal consideration by the ARG. Language Study Group proposals from the community should be for significant changes or new features that will require rigorous design and prototyping work. 
 
-Language Study Groups work towards the development of an AI that can be submitted to the ARG for formal consideration.
+Requests to form a new Language Study Group can be submitted through the official ARG [web-form](https://arg.adaic.org/community-input).
 
-AIs that are being being active considered by the ARG will be posted in this repository in the AI format. This is great reference for language study groups to understand how AIs should be formed, and what they should contain.
+It is generally encouraged that Language Study Groups be formed of group of interested individuals, but that is not a formal requirement.
 
-As the ARG deliberates on open AIs, AIs will be updated with changes and additional discussion.
+It is strongly encouraged that new ideas be put to as much community discussion as possible via a Github Issues (this repository) before proposing the formation of a new Language Study Group.
 
-At the present time, AIs are being stored in the Rich Text Format, with an aim to eventually develop tooling to easily convert it into many formats. RTF was selected due to its maturity, stability, and generality. The ARG will continue to review the AI formats in the future.
+When a new Language Study Group is approved or required by the ARG, a Google Docs shared folder will be created. This folder should remain publicly readable, and should contain all working documents used by the Study Group to carry-out its work.
 
-An AI template can be found in the root of this repository.
+A list of active Study Groups can be found [here](https://arg.adaic.org/community-input).
 
 ## Pull Requests
 
-Please do not make pull requests to this repository! AIs shall be submitted as the final steps of a Language Study Group. New AIs will be added to this repository by the ARG from time to time.
-
-
-
+Please do not make pull requests to this repository! AIs shall be submitted as the final steps of a Language Study Group. New AIs will be added to this repository by the ARG from time to time.
```

Comment:
"AIs shall be submitted ..." ==> "AIs should be submitted ..."

Reply:
It occurs to me that this entire paragraph is problematic. This was written at an earlier time when we were considering generating RTF and Markdown versions of AIs that we would also (redundantly) store on this repo. I've come up with this re-writing:

> Please do not make pull requests to this repository! The purpose of this repository is to enable discussion via the Github Issues feature. Please refer to the [ARG website](arg.adaic.org) for a listing of all active AIs and Language Study Groups.

Reply:
I still plan to do that (specifically HTML versions to duplicate the Ada-Auth.org ones), but it's behind a lot of other projects at this point. (And Markdown is contingent on someone converting the HTML version to Markdown; I know Richard was once working on that.)

Reply:
If it wasn't clear, I'd leave the sentence. The reason for putting copies here is that the version control is currently on my private systems -- carefully backed up to multiple locations and easily restorable, but not easily run by someone else. Eventually, everything will have to transition elsewhere, might as well get started.

Reply:
Improved this paragraph, leaving the possibility of adding AIs later.